### PR TITLE
Bump dev dependencies

### PR DESCRIPTION
**Pull request by DirtyF**
Q: Any possibility to keep this in sync with Jekyll?

### CODE REVIEW

**Comment by DirtyF** [comment]
@pathawks 🙁

File: jekyll-seo-tag.gemspec [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
+  spec.require_paths = ["lib"]
 
-  spec.add_dependency 'jekyll', '~> 3.3'
-  spec.add_development_dependency 'bundler', '~> 1.10'
-  spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'rspec', '~> 3.3'
-  spec.add_development_dependency 'html-proofer', '~> 2.5'
-  spec.add_development_dependency 'rubocop', '~> 0.37'
+  spec.add_dependency "jekyll", "~> 3.3"
+  spec.add_development_dependency "bundler", "~> 1.14"
```
Comment (pathawks) [inline jekyll-seo-tag.gemspec]: Is there a reason for us to _not_ allow `bundler 1.10`? The pessimistic operator will allow anything up to (but not including) `2.0`, so we only want to increment this if we have a reason to cut off support for a particular version.

Comment (DirtyF) [inline jekyll-seo-tag.gemspec]: 1.14 fixed an annoying bug in bundler https://github.com/bundler/bundler/issues/5154

**Comment by benbalter** [comment]
> Any possibility to keep this in sync with Jekyll?

What do you mean?

**Comment by DirtyF** [comment]
@benbalter When a plugin shares dependencies with jekyll, can it inherit from those? For instance `jekyll-seo-tag` depends on `bundler` like `jekyll`, does rubygems allows to say that `jekyll-seo-tag `and `jekyll` should share the same `bundler` version?

**Pull request merged**